distribution/docs/src/main/resources/_contents/_data/isr-attributes-table-contents.adoc

```diff
@@ -0,0 +1,344 @@
+
+.[[_isr_attributes_table]]ISR: Attributes in this group relate to intelligence, search, and reconnaissance.
+[cols="1,2,1,1,1" options="header"]
+|===
+|Term
+|Definition
+|Datatype
+|Constraints
+|Example Value
+
+|isr.frequency-hertz
+|A frequency observation, typically of radio waves.
+|Float
+|normalized to Hz
+|
+
+|isr.target-id
+|A target identifier.
+|String
+|< 1024 characters
+|
+
+|isr.target-category-code
+|A STANAG 3596 target category.
+|String
+|One of the values enumerated in STANAG 3596
+|
+
+|isr.platform-id
+|A platform identifier.
+|String
+|< 1024 characters
+|
+
+|isr.original-source
+|A STANAG 4545 ISOURCE.
+|String
+|ISOURCE per STANAG 4545
+|
+
+|isr.organizational-unit
+|An organizational unit.
+|String
+|< 1024 characters
+|
+
+|isr.niirs
+|The quality or detail level of an image. (National Imagery Interpretability Rating Scale)
```

Comment:
Should all these definitions end in a period? Seems like we are inconsistent throughout the tables on it.